tests/v1/kv_connector/nixl_integration/toy_proxy_server.py

```diff
@@ -10,10 +10,6 @@
 from fastapi import FastAPI, Request
 from fastapi.responses import StreamingResponse
 
-from vllm.logger import init_logger
```

Comment:
Instead of removing this, can we do:

import logging
 
logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)